src/main/scala/Covid19/Sources/RussianSource.scala

```diff
@@ -0,0 +1,32 @@
+package Covid19.Sources
+
+import Covid19.Protocol.{CovidData, Response}
+import cats.effect.IO
+import io.circe.Decoder
+import io.circe.parser.decode
+import sttp.client.{Identity, NothingT, SttpBackend, basicRequest}
+import sttp.client._
+
+final class RussianSource(implicit backend: SttpBackend[Identity, Nothing, NothingT], decoder: Decoder[Response]) extends Source {
```

Comment:
there is no needs to have the implicit decoder parameter

Reply:
why?